app/components/CategoryHeader.android.js

```diff
@@ -10,18 +11,39 @@ const styles = StyleSheet.create({
   },
 });
 
-const CategoryHeader = ({ category }) => (
-  <ToolbarAndroid
-    title={category.name}
-    style={styles.toolbar}
-    titleColor="white"
-  />
-);
+class CategoryHeader extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      icon: null,
+    };
+  }
+
+  componentWillMount() {
+    Icon.getImageSource('bars', 22, colors.headerText).then(source => this.setState({ icon: source }));
```

Comment:
Or even use https://github.com/oblador/react-native-vector-icons#toolbarandroid this component